tests/functional/cli/kvs/create/name-invalid-chars.sh

```diff
@@ -2,8 +2,10 @@
 
 # SPDX-License-Identifier: Apache-2.0
 #
-# Copyright (C) 2021 Micron Technology, Inc. All rights reserved.
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
 
 . common.subr
 
+trap remove_home EXIT
+
 cmd -e hse kvs create "$home" kvs!!
```

Comment:
I'd put quotes around the kvs name on line 11 since it has magic shell characters.

Reply:
I see. Yea I will change that and re-push